File: 3227-find-missing-and-repeated-values/3227-find-missing-and-repeated-values.js
/**
 * @param {number[][]} grid
 * @return {number[]}
 */
var findMissingAndRepeatedValues = function(grid) {
    let n = grid.length
    const square = Array(n * n + 1).fill(0)
    for (let i = 0; i < grid.length; ++i) {
        for (let j = 0; j < grid[i].length; ++j)
            square[grid[i][j]]++
    }
    const arr = Array(2).fill(0);
    for (let i = 0; i < square.length; ++i) {
        if (square[i] === 2)
            arr[0] = i
        if (square[i] === 0)
            arr[1] = i
    }
    return arr
};

File: 3227-find-missing-and-repeated-values/3227-find-missing-and-repeated-values.py
class Solution(object):
    def findMissingAndRepeatedValues(self, grid):
        """
        :type grid: List[List[int]]
        :rtype: List[int]
        """
        c = Counter(x for r in grid for x in r)
        d = {c[i]: i for i in range(1, len(grid) ** 2 + 1)}
        return [d[2], d[0]]